crates/pallets/starknet/src/lib.rs

```diff
@@ -201,22 +201,50 @@ pub mod pallet {
     pub(super) type StorageView<T: Config> =
         StorageMap<_, Twox64Concat, ContractStorageKeyWrapper, StarkFeltWrapper, ValueQuery>;
 
+    /// The last processed Ethereum block number for L1 messages consumption.
+    /// This is used to avoid re-processing the same Ethereum block multiple times.
+    /// This is used by the offchain worker.
+    /// # TODO
+    /// * Find a more relevant name for this.
```

Comment:
LastConsumedL1Block ?

Reply:
yes something like this. but will do it in a follow up PR, I just left the todo not to forget it.